src/trace_processor/importers/common/process_tracker.cc

```diff
@@ -306,19 +306,21 @@ UniquePid ProcessTracker::StartNewProcessInternal(
   // we keep them in |tids_| (if they weren't erased by EndThread()), but ignore
   // them in GetThreadOrNull().
 
+  // Same pid is never used concurrently by multiple processes, therefore remove
```

Comment:
Can you explain why this is safe to do? See comment literally above? Maybe remove the comment if it no longer applies?

Reply:
Shouldn't this maybe be conditional on is_main_thread?